ClavaWeaver/resources/clava/test/api/PassCompositionTest.js

```diff
@@ -1,45 +0,0 @@
-laraImport("clava.pass.DecomposeDeclStmt");
```

Comment:
Keep file, this is an important test

Reply:
Re-added in 4612a1f3b and refactored to remove erroneous code in ef6db7680.